README.md

```diff
@@ -3,8 +3,10 @@
 ## Supported Platforms
 
 * [AWS](docs/user/aws/README.md)
+* [Bare-metal (experimental)](docs/user/metal/install_upi.md)
```

Comment:
Could this list distinguish between UPI and IPI? At some point, we'll contribute baremetal IPI back to openshift/installer from Metal<sup>3</sup>, so there would be two options for it.

Reply:
> Could this list distinguish between UPI and IPI?

Do we need to do that here?  I'd expect to distinguish in the per-platform docs.

Reply:
Ah sure, I guess that works too.